src/OrbitGl/AccessibleTrack.cpp

```diff
@@ -146,12 +146,21 @@ AccessibilityState AccessibleTrackTab::AccessibleState() const {
   return AccessibilityState::Normal;
 }
 
-int AccessibleTrack::AccessibleChildCount() const { return 2; }
+int AccessibleTrack::AccessibleChildCount() const {
+  int child_count = track_->GetVisibleChildren().size();
+  return child_count + 2;
+}
 
 const AccessibleInterface* AccessibleTrack::AccessibleChild(int index) const {
-  if (index == 0) {
+  int child_count = track_->GetVisibleChildren().size();
```

Comment:
As discussed, this may be the reason for the Toggle not always showing up - the tab is "on top" of the toggle as they overlap. Either report them in a different order, or put the toggle as child of the tab.